### DISCUSSION TITLE
What are the benefits of using ChistaDATA Cloud?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#2] [❓FAQ] [answered]
What are the benefits of using ChistaDATA Cloud?

**Comment by TheraniAA** [comment, marked as answer]
ChistaDATA DBaaS manages the infrastructure administration tasks associated with ClickHouse®, saving time and effort. It monitors resource usage, automatically creates database backups, offers fault tolerance through automatic failover to backup replicas, and keeps the database software up to date. Users interact with database clusters in Managed Service for ClickHouse® in the same way as they would with regular databases in their local infrastructure.
